### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by Subhangini**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
```
Comment (himanshu007-creator) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: Delete line 18-23. Instead ask to explain the testing instructions.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<!---Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration-->
+
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration**:
```
Comment (himanshu007-creator) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: delete lines 24-28. Instead Make user input his machine configs:-

Operating system:
Version:
Text-editors used:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+* Firmware version:
+* Hardware:
+* Toolchain:
+* SDK:
+
+# Checklist:
```
Comment (himanshu007-creator) [inline .github/PULL_REQUEST_TEMPLATE.md:30]: After this list insert in H1 tags to " ATTACH SCREEN-SHOTS"

**Review by himanshu007-creator — CHANGES_REQUESTED** [review]
Make required changes

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (Subhangini) [reply .github/PULL_REQUEST_TEMPLATE.md:30]: ok

**Review by himanshu007-creator — APPROVED** [review]

**Pull request merged**